### DISCUSSION TITLE
Adding Manual Mesh Level To Compiled FW

### DISCUSSION THREAD

**Discussion by Cal-LF** [Jyers/Marlin#109] [Q&A] [answered]
I have been making different flavors of your firmware for the V2. I like having filament change and other options. How can I add Manual Mesh Level to my compiled FW?

**Comment by Jyers** [comment, marked as answer]
Try starting with one of the Manual Mesh templates and make any changes you need from there.

**Reply by Cal-LF** [reply]
Thanks, that works. Got it compiled on the first try. Now have filament change option.
